src/createServer.js

```diff
@@ -1,8 +1,51 @@
 'use strict';
 
+const http = require('http');
+const fs = require('fs');
+const url = require('url');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  const server = http.createServer((req, res) => {
+    const { pathname } = new url.URL(req.url, `http://${req.headers.host}`);
+    const filename = pathname.replace('/file/', '') || 'index.html';
```

Comment:
The replacement logic here assumes that the path will always start with '/file/'. Ensure that this assumption holds true in all cases, or add additional checks to handle unexpected paths.